omigami/utilities/gnps_helper.py

```diff
@@ -0,0 +1,25 @@
+import requests
+import pandas as pd
+
+
+class GnpsHelper():
+
+    def __init__(self):
+        pass
+
+
+    @staticmethod
+    def fetch_metadata(spectrum_id: str) -> pd.DataFrame:
+        """
+        Obtains spectrum metadata from GNPS using a spectrum ID.
+        """
+
+        gnps_metadata = requests.get(
+            "https://gnps.ucsd.edu/ProteoSAFe/SpectrumCommentServlet?SpectrumID="
+            + str(spectrum_id)
+        ).json()
+        gnps_metadata_df = pd.DataFrame(
+            eval(gnps_metadata["spectruminfo"]["peaks_json"]), columns=["m/z", "Intensity"]
+        )
+
+        return gnps_metadata_df
```

Comment:
I/O logic sounds like gateway not "Helper". **But**:
- there is only one method here
- it is used for plotting (not core use of the package)
- I don't see we adding more logic to a "GNPSGateway"

Therefore I think this can be a function in `omigami.plotting.utils`

Reply:
Ok, I will reconsider this class